azext_iot/tests/helpers.py

```diff
@@ -281,6 +288,44 @@ def clean_up_iothub_device_config(
             )
 
 
+def create_test_cert(
+    tracked_certs: List[str],
+    subject: str = "aziotcli",
+    cert_only: bool = True,
+    file_prefix: Optional[str] = None
+) -> str:
+    """
+    Creates a test certificate and appends it to the tracked certificates while returning the thumbprint.
+    Will create key certificate if specified.
+    """
+    output_dir = os.getcwd()
+    thumbprint = create_self_signed_certificate(
+        subject=subject,
+        valid_days=1,
+        cert_output_dir=output_dir,
+        cert_only=cert_only,
+        file_prefix=file_prefix,
+        sha_version=256,
+    )["thumbprint"]
+    name = file_prefix or subject
+    tracked_certs.append(name + CERT_ENDING)
+    if not cert_only:
+        tracked_certs.append(name + KEY_ENDING)
+    return thumbprint
+
+
+def set_cmd_auth_type(command: str, auth_type: str, cstring: str) -> str:
```

Comment:
This is also defined in .../tests/iothub/__init__.py , why not remove that implementation and reference this one where needed to reduce code duplication.